posthog/tasks/update_cache.py

```diff
@@ -58,8 +58,10 @@ def update_cache_item(key: str, cache_type: CacheType, payload: dict) -> List[Di
     filter = get_filter(data=filter_dict, team=team)
 
     # Doing the filtering like this means we'll update _all_ Insights with the same filters hash
-    dashboard_items = Insight.objects.filter(team_id=team_id, filters_hash=key)
-    dashboard_items.update(refreshing=True)
+    insights = Insight.objects.filter(
+        Q(team_id=team_id, filters_hash=key) | Q(team_id=team_id, id__in=DashboardTile.objects.filter(filters_hash=key))
```

Comment:
this `id__in`  seems wrong... it's comparing a list of insight ids with a queryset of `DashboardTile`

need to write a test/inspect the SQL to make sure this is behaving as expected